### ISSUE TITLE
Create "complete" user in fake data for development

### ISSUE THREAD

**Issue by mattbk** [gratipay/gratipay.com#3959]
This would make it quicker to get up to speed during development if `make fake` is used. 

Probably blocked on https://github.com/gratipay/gratipay.com/issues/3837.

**Comment by aandis** [comment]
Duplicate of #3960.

**Comment by mattbk** [comment]
Oops, submission glitch.
